content/attacks/posts/2023-04-09-GDAC.md

```diff
@@ -0,0 +1,42 @@
+---
+date: 2023-04-09
+target-entities: GDAC
+entity-types:
+  - Custodian
+  - Exchange
+attack-types: Wallet Hack
+title: "GDAC Hacked for $13 Million"
+loss: 13000000
+---
+
+## Summary
+
+On April 9, 2023, South Korean cryptocurrency exchange GDAC was hacked, resulting in the theft of cryptocurrencies worth approximately $13 million. Hackers gained access to the exchange's hot wallets and stole various cryptocurrencies, including Bitcoin (BTC), Ethereum (ETH), and Wemix (WEMIX).
+
+## Attackers
+
+The attackers behind the GDAC hack remain unidentified.
+
+GDAC Hacker ETH wallets:
+- [0x244615D99684175d31369332039b2D84ce925EC5](https://etherscan.io/address/0x244615D99684175d31369332039b2D84ce925EC5)
+- [0x62B5eb2cb925Ce2898f9327B235b3228e7Cac1C2](https://etherscan.io/address/0x62B5eb2cb925Ce2898f9327B235b3228e7Cac1C2)
+- [0x87597bDB421482190e223aCa0A4DEAd75AB0a98D](https://etherscan.io/address/0x87597bDB421482190e223aCa0A4DEAd75AB0a98D)
+- [0x6225c189f583fc2e17f5be36a6a9db4d3978872d](https://etherscan.io/address/0x6225c189f583fc2e17f5be36a6a9db4d3978872d)
+- [0x57192cca8b8e4beb77f3466c6d0550e64cc53b0f](https://etherscan.io/address/0x57192cca8b8e4beb77f3466c6d0550e64cc53b0f)
+
+## Losses
+
+GDAC lost approximately $13 million:
+- 10,000,000 WEMIX
+- 220,000 USDT
+- 350 ETH
+- 60.8 BTC
+
+## Timeline
+
+- **April 08, 2023, 06:36:59 PM +UTC:** [The exploiter starts to attack](https://etherscan.io/tx/0x242fea7585152bc1652aac4dbe9db3e8ceaa428f5372e22d40071ee2b50733a7)
+- **April 10, 2023:** [GDAC announced the hack on their platform.](https://support.gdac.com/hc/ko/articles/17389345782041--%EA%B8%B4%EA%B8%89%EA%B3%B5%EC%A7%80-%EC%9E%85%EC%B6%9C%EA%B8%88-%EC%84%9C%EB%B9%84%EC%8A%A4-%EC%9D%BC%EC%8B%9C%EC%A4%91%EB%8B%A8-%EB%B0%8F-%EA%B8%B4%EA%B8%89-%EC%84%9C%EB%B2%84%EC%A0%90%EA%B2%80-%EC%A7%84%ED%96%89-%EC%82%AC%EC%9C%A0)
```

Comment:
Fix source link please

Reply:
There is only a Korean article and the link contains Korean letters

Reply:
I tried remove the letters after digits like this: [https://support.gdac.com/hc/ko/articles/17389345782041](https://support.gdac.com/hc/ko/articles/17389345782041) and it still refers to the same page, so it's unnecessary